### DISCUSSION TITLE
Using CompreFace without API key, UI Panel and Postgres

### DISCUSSION THREAD

**Discussion by djordje-mijatovic** [exadel-inc/CompreFace#921] [Q&A]
Is there a way to user CompreFace end point (for example endpoints of VerifyController) without API key and Postgres storing the data? We just need a simple thing: We want to send 2 images to CompreFace (to VerifyController#verifyBase64 endpoint) and get the VerifyFacesResponse without using API key, Admin UI and other things. Is there a way to do it? 
Thank you

**Comment by pospielov** [comment]
Unfortunately no. 
The closest approach is to use the default verification API key `00000000-0000-0000-0000-000000000004`, but you will still need Postgres. And you will need to run the admin module once, as the DB initialization is in the admin module.
